config/log4j.properties

```diff
@@ -26,5 +26,5 @@ log4j.appender.PublicAccessLog.File=${ambry.log.dir}/publicAccessLog.out
 log4j.appender.PublicAccessLog.layout=org.apache.log4j.PatternLayout
 log4j.appender.PublicAccessLog.layout.conversionPattern=%m%n
 
-log4j.logger.PublicAccessLogger = INFO, PublicAccessLog
+log4j.logger.PublicAccessLogger = ERROR, PublicAccessLog
```

Comment:
INFO logging is generally unnecessary, and when needed may be enabled. The logs can otherwise grow considerably if a cluster with multiple AmbryServers are instantiated due to replication requests and responses filling them up.